src/plugin/regions/index.js

```diff
@@ -123,6 +123,9 @@ export default class RegionsPlugin {
         observerPrototypeKeys.forEach(key => {
             Region.prototype[key] = this.util.Observer.prototype[key];
         });
+
+        Region.prototype["_disabledEventEmissions"] = [];
```

Comment:
Thanks for the quick fix! Could you change the quotes to single?

Reply:
Why set `disabledEventEmissions` on region? It should be set on wavesurfer itself only. This is not a fix.

Reply:
no, this is the fix.  I had previously opened a PR where it wasn't the fix.

Reply:
@cadavre the code is confusing, but my fix is correct.  On line 123 we grab all the methods from `utils.Observer` and stuff them into `Region`.  but `Observer` now requires that `this` contains an array called `_disabledEventEmissions`.   If you read further you'll see that many objects in the wavesurfer codebase inherit from `Observer`, not just wavesurfer.

Reply:
@osheroff I see now – Plugins are Observers as well (`Plugin extends Observer`).

The `_disabledEventEmissions` was missing, because it's initialized in Observer's constructor. Shouldn't we call super() somewhere while initializing plugins @katspaugh ?

Reply:
Thanks for clarification @osheroff !

Reply:
> shouldn't we call super() somewhere while initializing plugins 

The Region plugin doesn't literally inherit from `Observer`, it kinda just borrows all the code.  I'm not sure why this was done, I think probably because they're built into different source files?  I agree it's kind of goofy what the Region plugin is doing here.